qt/aqt/data/web/pages/BUILD.bazel

```diff
@@ -13,6 +14,7 @@ copy_files_into_group(
 copy_files_into_group(
     name = "congrats_page",
     srcs = [
+        "congrats-base.css",
```

Comment:
Maybe it would be simpler to import the base file in index.ts on the svelte page, so it gets bundled into congrats.css?

```patch
diff --git a/ts/congrats/index.ts b/ts/congrats/index.ts
index 547b04c37..83d934114 100644
--- a/ts/congrats/index.ts
+++ b/ts/congrats/index.ts
@@ -4,7 +4,7 @@
 import { getCongratsInfo } from "./lib";
 import { setupI18n, ModuleName } from "anki/i18n";
 import { checkNightMode } from "anki/nightmode";
-
+import "./congrats-base.css";
 import CongratsPage from "./CongratsPage.svelte";

 export async function congrats(target: HTMLDivElement): Promise<void> {
```

Reply:
Hm, I think we have to differentiate between the component on the one hand, and the view (page) where it's used on the other (Even though the components are called CongratsPage and StatsPage :-) ). With congrats it's maybe not that dire, but for graphs it would probably affect my add-on including it in other views.

Reply:
I find the distinction a bit tricky to follow at the moment, but that may be my inexperience talking. :-)

Reply:
Let's say we wanted to turn the `toolbar` and `toolbar-bottom` into Svelte components, and include them into a new view including the congrats page. Now if all bundled the base css into their respective CSS files, we would have included (most-likely) the identical CSS three times by that time, because the components pack the CSS that is intended for the page.

Reply:
Thanks, that makes it clearer